library/ssl_tls.c

```diff
@@ -10045,6 +10052,9 @@ int mbedtls_ssl_get_key_exchange_md_tls1_2( mbedtls_ssl_context *ssl,
     const mbedtls_md_info_t *md_info = mbedtls_md_info_from_type( md_alg );
     *hashlen = mbedtls_md_get_size( md_info );
 
+    MBEDTLS_SSL_DEBUG_MSG( 2, ( "Perform mbedtls-based computation of digest \
```

Comment:
My intention in asking you to introduce this was to check for it in `ssl-opt.sh`; could you add the corresponding `-c XXX` line to the PSA tests?